src/routes/ui.svelte

```diff
@@ -154,76 +173,40 @@
     </div>
   </Card>
 
-  <!--todo: radios-->
   <Card id="radios">
```

Comment:
Page is too big. Why not dividing on containers?

Reply:
In some near future